api/client.php

```diff
@@ -308,6 +310,56 @@ function register() {
         exit;
     }
 
+    function edituser() {
+
+        $sex=@$_POST['sex'];
+        $qq=@$_POST['qq'];
+        $mail=@$_POST['mail'];
+        $from=@$_POST['from'];
+        $intro=@$_POST['intro'];
+        $hobby=@$_POST['hobby'];
+        $sig1=@$_POST['sig'];
```

Comment:
yes
refer to jiekouapi line394
$sig1=mysql_real_escape_string(@$_REQUEST['sig1']);